securedrop_client/gui/widgets.py

```diff
@@ -215,6 +215,9 @@ def __init__(self):
 
         self.active = False
 
+        self.download_animation = load_movie("sync_animation.gif")
```

Comment:
as discussed during standup you can just remove all these widget.py changes and focus on optimizing the UI so that it doesn't freeze when there are many sources by removing all the unnecessary update_sources calls and increasing the default timeout a bit.